tx/textual/internal/testdata/ledger.json

```diff
@@ -0,0 +1,91 @@
+		"expected": [
+			"0 | Chain id: my-chain",
+			"1 | Account number: 1",
+			"2 | Sequence: 2",
+			"3 | Public key: cosmos.crypto.secp256k1.PubKey",
+			"4 | > PubKey object",
+			"5 | >> Key: Auvdf+T963bciiBe9l15DNMOijdaXCUo6zqSOvH7TXlN",
+			"6 | Transaction: 1 Messages",
+			"7 | > Message (1/1): cosmos.bank.v1beta1.MsgSend",
+			"8 | > From address: cosmos1ulav3hsenupswqfkw2y3sup5kgtqwnvqa8eyhs",
+			"9 | > To address: cosmos1ejrf4cur2wy6kfurg9f2jppp2h3afe5h6pkh5t",
+			"10 | > Amount: 10 ATOM",
+			"11 | End of Messages",
+			"12 | Memo: > \u269B\\u269B\u269B     @",
+			"13 | Fee: 0.002 ATOM",
+			"14 | Gas limit: 100'000",
+			"15 | [1/2] Hash of raw bytes: e237dc2e3f8f0b1d0310e1cd0b0ab5fc4a59",
+			"16 | [2/2] e4bc3454d67eb65c3fdb0fa599c9"
+		],
+		"expert": false
+	},
+	{
+		"name": "long text in nested value",
+		"tx": "90a10172436861696e2069643a206d792d636861696ea101714163636f756e74206e756d6265723a2031a1016b53657175656e63653a2032a201782a5075626c6963206b65793a20636f736d6f732e63727970746f2e736563703235366b312e5075624b657903f5a3016d5075624b6579206f626a656374020103f5a30178314b65793a2041757664662b54393633626369694265396c3135444e4d4f696a64615843556f367a71534f76483754586c4e020203f5a101775472616e73616374696f6e3a2031204d65737361676573a20178244d6573736167652028312f31293a20636f736d6f732e676f762e76312e4d7367566f74650201a2016e50726f706f73616c2069643a20310201a2017834566f7465723a20636f736d6f7331756c6176336873656e7570737771666b77327933737570356b677471776e76716138657968730201a201774f7074696f6e3a20564f54455f4f5054494f4e5f5945530201a20178544d657461646174613a20612076657279206c6f6e67206c696e65206f6620746578742c20706c6561736520636fc3b670657261746520616e6420627265616b20696e746f206d756c7469706c65206c696e65732e0201a1016f456e64206f66204d65737361676573a10171466565733a20302e303032207561746f6da20172476173206c696d69743a203130302730303003f5a201785348617368206f66207261772062797465733a206532333764633265336638663062316430333130653163643062306162356663346135396534626333343534643637656236356333666462306661353939633903f5",
+		"expected": [
+			"0 | Chain id: my-chain",
+			"1 | Account number: 1",
+			"2 | Sequence: 2",
+			"3 | Public key: cosmos.crypto.secp256k1.PubKey",
+			"4 | > PubKey object",
+			"5 | >> Key: Auvdf+T963bciiBe9l15DNMOijdaXCUo6zqSOvH7TXlN",
+			"6 | Transaction: 1 Messages",
+			"7 | > Message (1/1): cosmos.gov.v1.MsgVote",
+			"8 | > Proposal id: 1",
+			"9 | > Voter: cosmos1ulav3hsenupswqfkw2y3sup5kgtqwnvqa8eyhs",
+			"10 | > Option: VOTE_OPTION_YES",
+			"11 | [1/2] > Metadata: Metadata: a very long line of text, please co\u00F6perate ",
+			"12 | [2/2] > and break into multiple lines. Also it ends in  a single ampersand @@",
```

Comment:
Ending `@` character is described in [annex2](https://github.com/cosmos/cosmos-sdk/blob/main/docs/architecture/adr-050-sign-mode-textual-annex2.md), but we can revisit if you have better ideas.